Diagnostics/AVTester/Test-ExchAVExclusions.ps1

```diff
@@ -337,32 +337,28 @@ while ($currentDiff -gt 0) {
             Write-Progress -Activity "Checking Exchange Processes" -CurrentOperation "$currentDiff More Seconds" -PercentComplete ((($initialDiff - $currentDiff) / $initialDiff) * 100) -Status " "
             [int]$currentDiff = (New-TimeSpan -End $StartDate.AddMinutes($WaitingTimeForAVAnalysisInMinutes) -Start (Get-Date)).TotalSeconds
 
-            # Determine if it is a known exchange process
-            if ($ProcessList -contains $process.path ) {
+            # Gather all modules
+            [array]$ProcessModules = $process.modules
 
-                # Gather all modules
-                [array]$ProcessModules = $process.modules
+            # Remove Microsoft modules
+            $ProcessModules = $ProcessModules | Where-Object { $_.FileVersionInfo.CompanyName -ne "Microsoft Corporation." -and $_.FileVersionInfo.CompanyName -ne "Microsoft" -and $_.FileVersionInfo.CompanyName -ne "Microsoft Corporation" }
 
-                # Remove Microsoft modules
-                $ProcessModules = $ProcessModules | Where-Object { $_.FileVersionInfo.CompanyName -ne "Microsoft Corporation." -and $_.FileVersionInfo.CompanyName -ne "Microsoft" -and $_.FileVersionInfo.CompanyName -ne "Microsoft Corporation" }
+            # Remove Oracle modules on FIPS
+            $ProcessModules = $ProcessModules | Where-Object { (-not($_.FileName -like "*\FIP-FS\Bin\*" -and $_.FileVersionInfo.CompanyName -eq "Oracle Corporation")) }
```

Comment:
This appears to be a GitHub display issue for some reason. When doing this via a git command, it shows up correctly.